cloudprem/modules/webhooks/main.tf

```diff
@@ -102,76 +92,41 @@ resource "aws_msk_cluster" "this" {
     }
   }
 
-//  logging_info {
-//    broker_logs {
-//      cloudwatch_logs {
-//        enabled   = true
-//        log_group = aws_cloudwatch_log_group.logs.name
-//      }
-//      s3 {
-//        enabled = true
-//        bucket  = aws_s3_bucket.bucket.id
-//        prefix  = "logs/msk-"
-//      }
-//    }
-//  }
-
   tags = var.tags
 }
-resource "aws_docdb_subnet_group" "default" {
-  name       = "${var.name}-mongo-subnet_group"
-  subnet_ids = var.subnet_ids
 
-  tags = var.tags
-}
-resource "aws_docdb_cluster_parameter_group" "disable_tls" {
-  family      = "docdb4.0"
-  name        = "${var.name}-mongo-param-grop"
-  description = "docdb cluster parameter group"
+resource "helm_release" "mongodb" {
 
-  parameter {
-    name  = "tls"
-    value = "disabled"
-  }
-}
-
-resource "aws_docdb_cluster_instance" "cluster_instances" {
-  count              = 1
-  identifier         = "${var.name}-mongo-${count.index}"
-  cluster_identifier = aws_docdb_cluster.this.id
-  instance_class     = "db.t3.medium"
+  name = "frontegg-documents"
+  repository = "https://charts.bitnami.com/bitnami"
+  chart = "mongodb"
+  version = "10.15.2"
 
-
-  tags = var.tags
+  set {
+    name = "auth.enabled"
+    value = "false"
+  }
 }
 
-resource "aws_docdb_cluster" "this" {
-  cluster_identifier = "${var.name}-mongo"
-  vpc_security_group_ids = [aws_security_group.this.id]
-  master_username    = "foo"
-  master_password    = "barbut8chars"
-  db_subnet_group_name = aws_docdb_subnet_group.default.name
-  skip_final_snapshot = true
-  db_cluster_parameter_group_name = aws_docdb_cluster_parameter_group.disable_tls.name
-
-  tags = var.tags
-}
+resource "helm_release" "redis" {
 
-resource "aws_elasticache_subnet_group" "default" {
-  name       = "${var.name}-redis-subnet-group"
-  subnet_ids = var.subnet_ids
-}
+  name = "frontegg-kvstore"
+  repository = "https://charts.bitnami.com/bitnami"
+  chart = "redis"
+  version = "14.1.1"
```

Comment:
Did frontegg struggle with managed redis? We're pull a _lot_ of storage and state into the cluster, which could be problematic. I suspect "small" (read: dev/QA) clusters might struggle to run kafka+mongo+redis?

I think we should prefer to use managed services whereever possible.. but i suspect it just didn't work with frontegg?

Reply:
Actually we would prefer not to use managed services for things like this. It's such a small use and requires no clustering or advanced features. Their use of redis is basically a queue store so there's no persistent data stored there so there is little risk of data loss. I didn't see a reason to add $20 or so to the monthly bill for no real gain but it did work fine with elasticache redis so if we wanted to use that instead we could. It did not, however, work with aws's mongo (documentDB). That requires a pure mongo server.

Reply:
Okay sure. My concern is more around running storage in cluster. So if these services start thrashing the cluster, they could effect the app critical path.

Also, i think we could see surprising behavior as we replace nodes. If we replace the whole node ASG, those pods are going to have to move around, _AND_ they're stateful, so they'll be down for quite a bit while the PVs move around. 

I think that's worth pointing out. It might allow for some dataloss of inflight webhook events during scaling..